client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -17,6 +17,7 @@ import { Diana } from '~/services/contributors';
 import Highlight from '~/components/Highlight/Highlight';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.